examples/README.md

```diff
@@ -3,27 +3,26 @@
 Tested with:
 
 
-cargo 0.11.0-nightly (3ff108a 2016-05-24)
+cargo 0.18.0-nightly (be0b499 2017-02-22)
 
-rustc 1.11.0-nightly (a967611d8 2016-05-30)
+rustc 1.17.0-nightly (413a975e3 2017-02-23)
 
-ruby 2.3.1p112 (2016-04-26 revision 54768)
+ruby 2.4.0p0 (2016-12-24 revision 57164)
 
-Bundler version 1.12.5
+Bundler version 1.14.4
 
 ### Main
 1. In the main project `cargo build --release`
 
 ### Ruby gem
 1. Go to helix/ruby
 2. `bundle install`
-  * [requires neversaydie](https://github.com/tenderlove/neversaydie)
+  * [requires neversaydie](https://github.com/tenderlove/neversaydie) (NOTE: `libsigsegv` is also available via HomeBrew)
```

Comment:
Let's change this to say `* Note: neversaydie (link to repo) requires libsigsegv (link to the #install section in the readme) ` + your note

...but the more I think about it, the tests that added this dep just doesn't seem worth it, I would propose that we remove it soon 😐